### PR TITLE
Move the window settings below the acrylic settings

### PR DESCRIPTION
**Pull request by zadjii-msft**
image

Put some settings below the acrylic & background image settings. Those groups have controls that will become visible when the user enables the setting. If there's other controls below them, then it's less likely that the user has _exactly scrolled to the checkbox_. That means it's more likely that the newly visible controls will be on-screen.

* [x] closes one checkbox in #8764
* [x] I work here

### CODE REVIEW

File: src/cascadia/TerminalSettingsEditor/Profiles.cpp [hunk 1]
```diff
@@ -277,4 +277,5 @@ namespace winrt::Microsoft::Terminal::Settings::Editor::implementation
     {
         return _State.Profile().CursorShape() == TerminalControl::CursorStyle::Vintage;
     }
+
```
Comment (carlos-zamora) [inline src/cascadia/TerminalSettingsEditor/Profiles.cpp:280]: nit: you can revert changes to this file

**Review by carlos-zamora — APPROVED** [review]

**Comment by cinnamon-msft** [comment]
This is clever, I dig it

**Review by cinnamon-msft — APPROVED** [review]

**Comment by zadjii-msft** [comment]
@msftbot merge this in 1 minute

**Comment by mdtauk** [comment]
Are Opacity settings disabled when Acrylic is turned off?

**Comment by zadjii-msft** [comment]
@mdtauk Yep, that's the whole point of what's getting fixed here:
> Check/uncheck Enable Acrylic box and the scroll bar dances (because the opacity slider appears off the screen to the bottom…. Which isn't obvious. Can we scroll it onto the screen?)

**Comment by mdtauk** [comment]
> @mdtauk Yep, that's the whole point of what's getting fixed here:
> 
> > Check/uncheck Enable Acrylic box and the scroll bar dances (because the opacity slider appears off the screen to the bottom…. Which isn't obvious. Can we scroll it onto the screen?)

Thank you, I was just trying to understand what was being discussed.

Grouping those controls which are directly affected by controls above, is a very sensible approach, across all settings pages.

**Pull request merged**